flecsi/run/context.hh

```diff
@@ -342,10 +344,12 @@ struct context {
       parsed.options, boost::program_options::include_positional);
 
 #if defined(FLECSI_ENABLE_FLOG)
+#if FLECSI_RUNTIME_MODEL != FLECSI_RUNTIME_MODEL_hpx
```

Comment:
@tuxfan: Could we just defer this initialization for _all_ backends to avoid the special case?

Reply:
I actually changed this back as for now I reverted the changes to the logging library.